ironfish-cli/scripts/import-export-test.sh

```diff
@@ -5,6 +5,8 @@ set -e # exit immediately if anything returns with non-zero exit code
 # Change working directory to the script's directory
 cd "$(dirname "$0")"
 
+DATA_DIR="../testdbs/importexport"
```

Comment:
nit - I think if you use `IRONFISH_DATA_DIR` here you won't need to set it in each command